### ISSUE TITLE
Test issue: No module named 'pytest_astropy'

### ISSUE THREAD

**Issue by olebole** [astropy/pytest-astropy-header#6]
I am just packaging this module for Debian, since it is a new dependency of pytest-astropy.
When I run the tests in the `tests` subdirectory, I get
```
testdir = <Testdir local('/tmp/pytest-of-ole/pytest-6/test_modify_in_conftest0')>
capsys = <_pytest.capture.CaptureFixture object at 0x7fcfa0bb2d50>

    def test_modify_in_conftest(testdir, capsys):
        testdir.makeconftest("""
        from pytest_astropy.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
    
        def pytest_configure(config):
            config.option.astropy_header = True
            PYTEST_HEADER_MODULES.pop('Pandas')
            PYTEST_HEADER_MODULES['scikit-image'] = 'skimage'
            TESTED_VERSIONS['fakepackage'] = '1.0.2'
        """)
        testdir.inline_run()
        out, err = capsys.readouterr()
        lines = extract_package_version_lines(out)
>       assert len(lines) == 6
E       assert 0 == 6
E        +  where 0 = len([])

/home/ole/Projects/2011/debian/python/astropy/pytest-header/tests/test_display.py:191: AssertionError
```
and when I print the `err` variable:
```
ImportError while loading conftest '/tmp/pytest-of-ole/pytest-6/test_modify_in_conftest0/conftest.py'.
conftest.py:1: in <module>
    from pytest_astropy.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
E   ModuleNotFoundError: No module named 'pytest_astropy'
```
That seems reasonable, since pytest-astropy does not have its own module but is just a metapackage with dependencies, right?
I tried with pytest-astropy 0.5 and 0.6.
Even if it would work, I would be a bit unhappy, since it creates a circular dependency in the tests (not a big problem, however).

**Comment by pllim** [comment]
There is an unmerged fix in #2

**Comment by olebole** [comment]
Thank you @pllim that solves the problem! Just uploading the package now...
